python/tk_nuke/__init__.py

```diff
@@ -82,27 +82,23 @@ def __create_tank_error_menu():
     
 def __engine_refresh(tk, new_context):
```

Comment:
Since `new_context` has a `tank` object as a member, we could use it directly instead of passing as an extra argument, no ?

Like [here](https://github.com/shotgunsoftware/tk-nuke/pull/58/files#diff-2082bc816b93dbd02e442c7942fa7709R88)